pywhyllm/suggesters/validation_suggester.py

```diff
@@ -93,29 +76,29 @@ def request_negative_controls(
-        model is a conceptual model that describes the causal mechanisms of a system. You will do this by answering 
-        questions about cause and effect using your domain knowledge in the {domain_expertise}."""
+            studying the {analysis_context}. You are using your domain knowledge to help understand the negative 
+            controls for a causal model that contains all the assumptions about the {analysis_context}. Where a causal 
+            model is a conceptual model that describes the causal mechanisms of a system. You will do this by answering 
+            questions about cause and effect using your domain knowledge in the {domain_expertise}."""
 
                 with user():
                     lm += f"""factor_names: {factors_list} From your
-                         perspective as an expert in the {domain_expertise}, what factor(s) from the list of factors, relevant to 
-                         the {analysis_context}, should see zero treatment effect when changing the {treatment}? Which factor(s) 
-                         from the list of factors, if any at all, relevant to the {analysis_context}, are negative controls on the 
-                         causal mechanisms that affect the {outcome} when changing {treatment}? Using your domain knowledge, 
-                         which factor(s) from the list of factors, if any at all, relevant to the {analysis_context}, 
-                         should we expect to be unaffected by any changes in {treatment}? Which factor(s) from the list of factors, 
-                         if any at all, would be surprising if affected by a change in {treatment}? Be concise and keep your 
-                         thoughts under two paragraphs. Then provide your step by step chain of thoughts within the tags 
-                         <thinking></thinking>. Once you have thought things through, wrap the name of the factor(s) from the list of 
-                         factors, that has/have a high likelihood of being negative controls on the causal mechanisms that affect {outcome}
-                         when changing {treatment}, within the tags <negative_control>factor_name</negative_control>. Wrap the name 
-                         of the factor(s) from the list of factors, that has/have a high likelihood of being unaffected when 
-                         changing {treatment}, within the tags <negative_control>factor_name</negative_control>. Where factor_name 
-                         is one of the items within the factor_names list. If a factor does not have a high likelihood of being a 
-                         negative control relevant to the {analysis_context}, then do not wrap the factor with any tags. Provide 
-                         your step by step answer as an expert in the {domain_expertise}:"""
+                             perspective as an expert in the {domain_expertise}, what factor(s) from the list of factors, relevant to 
```

Comment:
are these extra spaces intended? Would they add unnecessary spaces in the final prompt?